### ISSUE TITLE
A big delay when dragging

### ISSUE THREAD

**Issue by R0FGV** [QL-Win/QuickLook#540]
In the Windows 1903 system, there is a big delay when dragging a fixed picture window. Is it an individual phenomenon?

**Comment by xupefei** [comment]
Are you using Intel HD 4xxx GPU?

**Comment by vasiby** [comment]
> Is it an individual phenomenon?

I've seen several apps that use the new windows 10 acrylic transparent background to lag when dragging

**Comment by xupefei** [comment]
> > Is it an individual phenomenon?
> 
> I've seen several apps that use the new windows 10 acrylic transparent background to lag when dragging

Yeah it’s true. The bug seems alive even in the coming 1909 update. See a discussion here: https://github.com/sourcechord/FluentWPF/issues/42.

**Comment by LunarLanding** [comment]
Hi, I am trying to solve a lag issue with using QuickLook, and this issue seems like the best match to my issue.
The issue discussed in sourcechord/FluentWPF#42 seems fixed in both windows versions 1903 and 1909.
Is updating the version of FluentWPF a viable step, or a complicated one?

**Comment by emako** [comment]
Outdated issue
